packages/administration/translations/messages.sk.po

```diff
@@ -1936,6 +1936,9 @@ msgstr "Tento stav nemôže byť odstránený, pretože je naň viakaná funkcio
 msgid "This status can't be deleted, because there is a functionality bounded to it - it identifies orders that were successfully processed."
 msgstr "Tento stav nemôže byť odstránený, pretože je naň viakaná funkcionalita - identifikuje objednávky, ktoré boli úspešne spracované."
 
+msgid "This status can't be deleted, because there is a functionality bounded to it - it identifies orders with withdrawal requests."
+msgstr "Tento stav nemôže byť odstránený, pretože je naň viakaná funkcionalita - určuje objednávky s požiadavkou na odstúpenie od zmluvy."
```

Comment:
Isn't there a typo in the first part of the message? (same for other messages)

```
Tento stav nie je možné zmazať, pretože je naň naviazaná funkčnosť
```

Reply:
yes :sweat_smile: a315aa0264